charts/nfdiv-frontend/values.yaml

```diff
@@ -16,9 +16,10 @@ nodejs:
         - AppInsightsInstrumentationKey
         - redis-access-key
   environment:
-    IDAM_WEB_URL: "https://idam-web-public.{{ .Values.global.environment }}.platform.hmcts.net/login"
-    IDAM_API_URL: "https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net/o/token"
-    REDIS_HOST: "nfdiv-frontend-redis-{{ .Values.global.environment }}.redis.cache.windows.net"
+    IDAM_WEB_URL: 'https://idam-web-public.{{ .Values.global.environment }}.platform.hmcts.net/login'
+    IDAM_API_URL: 'https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net/o/token'
+    REDIS_HOST: 'nfdiv-frontend-redis-{{ .Values.global.environment }}.redis.cache.windows.net'
+    COS_URL: 'http://nfdiv-cos-{{ .Values.global.environment }}.service.core-compute-aat.internal'
```

Comment:
you will also need to update `values.preview.template.yaml` for `COS_URL` to have value `http://nfdiv-cos-aat.service.core-compute-aat.internal` assuming in preview you want to use AAT ?